ui/components/ui/info-tooltip/info-tooltip.js

```diff
@@ -49,10 +48,6 @@ InfoTooltip.propTypes = {
    * Shows position of the tooltip
    */
   position: PropTypes.oneOf(['top', 'left', 'bottom', 'right']),
-  /**
-   * Set if the tooltip wide
-   */
-  wide: PropTypes.bool,
   /**
    * Add custom CSS class for container
    */
```

Comment:
Did a search for `wide` prop and its only used in one instance that probably can be the same width as the other InfoTooltips

<img width="200" alt="Screen Shot 2022-06-20 at 1 37 59 PM" src="https://user-images.githubusercontent.com/8112138/174674822-9b90ff19-2fdf-486d-9987-0ec4ed7482ca.png">


<img width="300" alt="Screen Shot 2022-06-20 at 1 24 42 PM" src="https://user-images.githubusercontent.com/8112138/174674881-4c509cc7-2070-4529-a2b7-6c7e501f25ad.png">